backend/src/main/java/pl/krkteam/battleships/common/domain/Board.java

```diff
@@ -19,12 +18,16 @@ public Cell putCoordinatesAndCell(Coordinates coordinates, Cell cell) {
         return coordinatesCellMap.put(coordinates, cell);
     }
 
+    public Cell getCoorValue(Coordinates coordinates) {
+        return coordinatesCellMap.get(coordinates);
+    }
+
     public BoardSize getBoardSize() {
         return boardSize;
     }
 
-    public Map<Coordinates, Cell> getCoordinatesCellMap() {
-        return Collections.unmodifiableMap(coordinatesCellMap);
+    Map<Coordinates, Cell> getCoordinatesCellMap() {
+        return coordinatesCellMap;
```

Comment:
Should be still immutable (like it was).
If it needs to be modified, you should create separate method for it